packages/npm/@sentry/svelte/7.9.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Sentry Svelte SDK",
+  "canonical": "npm:@sentry/svelte",
+  "version": "7.9.0",
```

Comment:
I added version 7.9.0 which is the latest version of the JS SDKs. Should I bump this to the version that the Svelte SDK will be released with (which I suspect will be 7.10.0)?

Reply:
Yeah this should be the first version that the SDK is available with. Hence we should wait until that first release for us to add to the release registry.

Reply:
Cool, then let's wait for the release 👍